templates/pesto/src/recipe-app.html

```diff
@@ -118,21 +126,28 @@
 
           <!-- nav menu -->
           <paper-listbox
-              selected="{{_pageData.page}}"
+              selected="[[_pageData.page]]"
```

Comment:
Let's add a comment with a TODO that we should change this back once the issue in Polymer2 is fixed.